conference/react-js/src/Routers.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Upcomming from './components/Upcomming';
+import Past from './components/Past';
+
+const Routers = () => (
+  <Router>
+    <div className="Container home">
+      <ul className="tabs-list">
+        <li className="tabs-list__item">
+          <Link
```

Comment:
Use ```NavLink``` for ```Navigation```. It has property ```activeClassName```. Pass ```tabs-list__anchor_active``` to this property and ```Router``` handle active route for You.